restapi/src/test/java/io/stargate/web/docsapi/service/DocsShredderTest.java

```diff
@@ -0,0 +1,1237 @@
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "[000001]",
+          null,
+          null,
+          true
+        };
+    Object[] expectedRow3 =
+        new Object[] {
+          "documentId",
+          "nested",
+          "path",
+          "spot",
+          "array",
+          "[000002]",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "[000002]",
+          null,
+          3.0,
+          null
+        };
+    Object[] expectedRow4 =
+        new Object[] {
+          "documentId",
+          "nested",
+          "path",
+          "spot",
+          "nested",
+          "structure",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "structure",
+          null,
+          23.456,
+          null
+        };
+
+    return ImmutableList.of(expectedRow0, expectedRow1, expectedRow2, expectedRow3, expectedRow4);
```

Comment:
This is fine; JDK has `return Arrays.asList(expectedRow0, ... , expectedRow4);` alternative as well.